storage/migrations/13_parsed_message.go

```diff
@@ -0,0 +1,24 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+func init() {
+	up := batch(`
+CREATE TABLE IF NOT EXISTS public.message_parsed (
+	"cid" text NOT NULL,
+	"from" text NOT NULL,
+    "to" text NOT NULL,
+	"value" text NOT NULL,
+	"method" text NOT NULL,
+	"params" jsonb,
+	PRIMARY KEY ("cid")
+);
+`)
```

Comment:
We should think about indexing the params. If the JSON is fairly flat mostly key/value then we could get a lot of mileage from a GIN index like:

```
CREATE INDEX public.message_parsed_params_idx ON message_parsed USING GIN (params);
```

A good overview of some of the options: [Using JSONB in PostgreSQL](https://scalegrid.io/blog/using-jsonb-in-postgresql-how-to-effectively-store-index-json-data-in-postgresql/#6)

Reply:
It looks like a GIN index allows for fast filtering on presence of top-level keys in the json. I think we do better to put a direct index on `Method` - the keys for each message of a given method should be identical, so the filter on the GIN would be equivalent and still probably not as good as being able to filter by method, right?